client/slack.go

```diff
@@ -147,6 +147,20 @@ func (sl *slackClient) handleMessageEvents() {
 				continue
 			}
 			sl.incomingMessages <- toFlyteMessageEvent(v, u)
+
+		case *slack.ReactionAddedEvent:
+			u, err := sl.client.GetUserInfo(v.User)
+			log.Debug().Msgf("received reaction event for type = %v", v)
+			if err != nil {
+				log.Err(err).Msgf("cannot get info about user=%s: %v", v.User, err)
+				continue
```

Comment:
I believe usage of continue is based on the `MessageEvent`

I would say the most important functionality is to deliver event to the channel and if we fail to get user/itemUser this is secondary concern. It is better to deliver some data depending on how important it is to have user details. I would leave it down to flow to decide what to do with such data e.g. some flows will be happy and wont require user details, some might require it.

Lets remove it. It might have a knock off effect on some other places.

Reply:
I have removed this continue, I will let the flow handle the usecase , as suggested .